### DISCUSSION TITLE
【求助】绳匠们，在仙舟太卜司发现的，这是什么书啊

### DISCUSSION THREAD

**Discussion by RushiShuo-Max** [share121/inter-knot#1501] [其他]
就是这本书，书的年代感觉相当久远，我是偷偷闯进禁书阁的，当时不知道为什么就突然出现在我面前，（~~有些后怕~~），是不是什么妖书啊!
![capture_20240811213735682](https://github.com/user-attachments/assets/22303d62-be90-4dc3-b374-5311a73f9cab)

**Comment by Dave-12138** [comment]
你看见有uploading就说明图还没上传完，要等上传完再点提交

**Reply by RushiShuo-Max** [reply]
> 你看见有uploading就说明图还没上传完，要等上传
好的，谢谢🦀

**Comment by ZhuYuan-148** [comment]
不知道的失物可以移交给治安局调查。

**Comment by WORLD-executeME** [comment]
上交云骑军吧
